src/hooks/transfers/useTransfers.tsx

```diff
@@ -69,17 +69,19 @@ export const useTransfers = () => {
   const { dydxAddress, sourceAccount } = useAccounts();
   const selectedDydxChainId = useAppSelector(getSelectedDydxChainId);
 
+  // coinbase is the only exchange we currently support
+  // TODO: remove typecast once we add more exchanges
+  const [exchangeName, setExchangeName] = useState<'coinbase' | undefined>();
   const [fromTokenDenom, setFromTokenDenom] = useState<string | undefined>();
   const [fromChainId, setFromChainId] = useState<string | undefined>();
   const [toTokenDenom, setToTokenDenom] = useState<string | undefined>();
   const [toChainId, setToChainId] = useState<string | undefined>();
-  // TODO [onboarding-rewrite]: add nobleAddress type when enabling coinbase withdrawals
-  const [fromAddress, setFromAddress] = useState<EvmAddress | SolAddress | DydxAddress | undefined>(
-    undefined
-  );
-  const [toAddress, setToAddress] = useState<EvmAddress | SolAddress | DydxAddress | undefined>(
-    undefined
-  );
+  const [fromAddress, setFromAddress] = useState<
+    EvmAddress | SolAddress | DydxAddress | NobleAddress | undefined
+  >(undefined);
```

Comment:
```suggestion
  const [fromAddress, setFromAddress] = useState<
    EvmAddress | SolAddress | DydxAddress | NobleAddress>();
```

Reply:
i swear i fixed this so many times but it keeps getting rebased out 😢 will do 👍